make-nagios-fhservices-cfg

```diff
@@ -24,6 +24,13 @@ rhmap_admin_email = os.getenv('RHMAP_ADMIN_EMAIL', 'root@localhost')
 rhmap_router_dns = os.getenv('RHMAP_ROUTER_DNS', 'localhost')
 rhmap_hostgroups = os.getenv('RHMAP_HOSTGROUPS', 'core')
 
+if rhmap_hostgroups == 'mbaas':
+    mongodb_count = int(os.getenv("MONGODB_ENDPOINT_COUNT", 3))
```

Comment:
MONGODB_ENDPOINT_COUNT isn't being set in any template that we are using, so this will always be 3, is there any point checking this if we are never going to set it?